qutebrowser/extensions/loader.py

```
@@ -118,6 +121,42 @@ def _walk_normal() -> typing.Iterator[ExtensionInfo]:
         yield ExtensionInfo(name=name)
 
 
+def walk_extensions() -> typing.Iterator[ExtensionInfo]:
+    """Walk external extensions."""
+    ext_dir = os.path.join(standarddir.data(), 'extensions')
+    if not os.path.exists(ext_dir):
+        try:
+            os.mkdir(ext_dir)
+        except OSError:
+            pass
```

Is it a good idea to silently ignore all OSErrors here?